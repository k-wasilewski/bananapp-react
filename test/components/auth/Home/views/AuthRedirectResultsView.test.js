import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import AuthRedirectResultsView from "../../../../../src/components/auth/Home/views/AuthRedirectResultsView";
import {Provider} from "react-redux";

describe("AuthRedirectResultsView rendering specification", () => {
    it('AuthRedirectResultsView is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <AuthRedirectResultsView />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});