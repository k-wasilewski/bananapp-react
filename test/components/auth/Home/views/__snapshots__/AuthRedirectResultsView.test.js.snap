// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthRedirectResultsView rendering specification AuthRedirectResultsView is rendered 1`] = `null`;
